spec/spec_helper.rb

```diff
@@ -23,3 +23,7 @@ class Commentable < ActiveRecord::Base
 end
 
 ActiveRecord::Base.send(:include, Acts::CommentableWithThreading)
+
+RSpec.configure do |config|
+  config.expect_with(:rspec) { |c| c.syntax = :should }
+end
```

Comment:
The gem uses the old rspec syntax, so to minimize the diff, we set the `:should` syntax as the preferred choice.